### PR TITLE
vendor: update vishvananda/netlink v1.1.0

### PR DESCRIPTION
**Pull request by thaJeztah**
full diff: https://github.com/vishvananda/netlink/compare/v1.0.0...v1.1.0

also updated moby/ipvs, which is compatible with this version of netlink,
and update vishvananda/netns to current master (which added go.mod)

### CODE REVIEW

**Comment by thaJeztah** [comment]
ping @arkodg @cpuguy83 PTAL

File: vendor.conf [hunk 1]
```diff
@@ -58,4 +58,4 @@ go.opencensus.io                        9c377598961b706d1542bd2d84d538b5094d596e
 gotest.tools                            1083505acf35a0bd8a696b26837e1fb3187a7a83 # v2.3.0
 github.com/google/go-cmp                3af367b6b30c263d47e8895973edcca9a49cf029 # v0.2.0
 
-github.com/moby/ipvs                    45bd6e080830fe2f2a43320b154d82b8f3f5aa0c
+github.com/moby/ipvs                    8f137da6850a975020f4f739c589d293dd3a9d7b
```
Comment (thaJeztah) [inline vendor.conf]: This was now tagged as v1.0.0; let me add a comment and push

**Comment by cpuguy83** [comment]
Known failure?
```
ERRO[0032] node: 178c1d58c041 is unknown to memberlist  
--- FAIL: TestNetworkDBCRUDMediumCluster (8.84s)
    networkdb_test.go:117: Network existence verification failed
```

**Comment by thaJeztah** [comment]
I think that one may be flaky; let me kick it again (it was green on the previous run)

**Comment by arkodg** [comment]
@thaJeztah its not really flaky
Ive seen these errors
```
Failed to join 172.17.0.2: dial tcp 172.17.0.2:10033: connect: connection refused
``` 
at times when we rerun the same job again in a short duration (before the previous one ends)

**Comment by thaJeztah** [comment]
@arkodg so is it a bug/regression in this PR?

**Comment by arkodg** [comment]
no the unit tests need to be made more robust to handle job reruns

**Comment by thaJeztah** [comment]
@cpuguy83 @arkodg it's green now 👍

**Pull request merged**